lib/BugYield/BugTracker/FogBugzBugTracker.php

```diff
@@ -1,201 +0,0 @@
-    // when reporting hours with decimals. Silly FogBugz.
-    $params['hrsElapsedExtra'] = number_format($elapsed_hours, 2, ',', '');
-
-    // Add the (updated) data to the FogBugz entry.
-    $request = new \FogBugz_Request($this->api);
-    $request->setParams($params);
-    $response = $request->go();
-
-    return $update;
-  }
-
-  public function deleteWorkLogEntry($worklogId) {
-    // NOT IMPLEMENTED IN FB
-    return false;
-  }
-
-  /**
-   * Fetch the Harvest data from the FogBugz updates. 
-   *
-   * @param FogBugz_Response_Cases $response
-   * @return Array Matches from the regex preg_match
-   */
-  private function getHarvestEntriesFromFBTicket(\FogBugz_Response_Cases $response, $include_overwritten_entries = FALSE) {
-    
-    $harvestEntries = array();
-
-    if(!isset($response->_data)) {
-      error_log("no data from this response");
-      return $harvestEntries;
-    }
-
-    foreach($response->_data as $case) {
-      $fbId = $case->_data['ixBug'];
-      if (isset($case->_data['events'])) {
-        //Reverse the order of events to get the most recent first.
-        //These will contain the latest updates in regard to time and task.
-        $events = $case->_data['events']->_data;
-        foreach ($events as $event) {
-
-          $text = (isset($event->_data['sHtml'])) ? $event->_data['sHtml'] : $event->_data['s'];
-          if (is_string($text) && preg_match('/^Entry\s#([0-9]+)\s\[(.*?)\/(.*?)\]:\s(.*?)by\s(.*?)@\s([0-9-]+)\sin\s(.*?)(\s\(updated\))?$/', $text, $matches)) {
-            $timelog = new \stdClass;
-            $timelog->harvestId = $matches[1];
-            $timelog->hours     = $matches[2];
-            $timelog->taskName  = self::convertAndCleanData($matches[3]);
-            $timelog->notes     = self::convertAndCleanData($matches[4]);
-            $timelog->user      = self::convertAndCleanData($matches[5]);
-            $timelog->spentAt   = $matches[6];
-            $timelog->project   = self::convertAndCleanData($matches[7]);
-            $timelog->updated   = isset($matches[8]) ? TRUE : FALSE;
-            $timelog->remoteId  = $event->_data['ixBugEvent'];
-            
-            if ($include_overwritten_entries) {
-              $harvestEntries[] = $timelog;
-            }
-            else {
-              $harvestEntries[$matches[1]] = $timelog;
-            }
-          }
-        }
-      }
-      
-    }
-    
-    return $harvestEntries;
-  }
-
-  /**
-   * Return the TicketID in its purest form, e.g. removing #hashmarks
-   */
-  public function sanitizeTicketId($ticketId) {
-    $ticketId = intval(str_replace("#","",$ticketId));
-    return $ticketId;
-  }
-
-  /**
-   * Fogbugz returns data in a wierd format. Convert and clean in up for easier comparison of strings
-   */
-  private function convertAndCleanData($data) {
-    $data = trim(trim(strip_tags(html_entity_decode(str_replace("&nbsp;"," ",$data), ENT_COMPAT, "UTF-8"))), '"');
-    return $data;
-  }
-}
```

Comment:
Consensus between me and @luckow was that we no longer use FogBugz and didn't really want to maintain the support of it anymore, so we simply removed it :)